Makefile

```diff
@@ -223,7 +223,7 @@ build-packages-python-standalone: ## Build Python packages (standalone)
 		--workdir=/app/build \
 		$(docker_development_image_repository):$(docker_image_version) \
 		/bin/bash -c "cmake -DBUILD_UNIT_TESTS=OFF -DBUILD_PYTHON_BINDINGS=ON .. \
-		&& $(MAKE) -j 4 \
+		&& $(MAKE) -j $(shell nproc) \
```

Comment:
This was hardcoded to 4 as it was causing issues in the pipeline by causing it crash if all processors were busy I believe.

Reply:
Ah I see, I'll revert

Reply:
> This was hardcoded to 4 as it was causing issues in the pipeline by causing it crash if all processors were busy I believe

Yes, let's keep the current numbers on nproc for now until we understand better the behavior 🙏 (I made that mistake before)